service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookHandler.java

```diff
@@ -53,4 +55,8 @@ public WsmResource makeResourceFromDb(DbResource dbResource) {
             .build();
     return resource;
   }
+
+  public String generateCloudName(@Nullable UUID workspaceUuid, String aiNotebookName) {
+    return aiNotebookName;
```

Comment:
Do we need to guarantee this is a valid cloud resource name, or is it ok if this endpoint returns a name GCP won't accept?